### DISCUSSION TITLE
Zwift on screen gear indication

### DISCUSSION THREAD

**Discussion by doudar** [cagnulein/qdomyos-zwift#1205] [Ideas]
Zwift shows gear indication for the Wattbike Atom, a Smart Bike connected by BLE. 

[Video by GP Lama shows it in action.](https://fb.watch/iezLEv0NWN/) 
![image](https://user-images.githubusercontent.com/17362216/214123495-9084bdfd-d8ba-47f2-bee9-6bdc1601fb8a.png)

We should see if we can get a log from NRF Connect from someone that owns one of these so that we can expose the same data to Zwift.

**Comment by cagnulein** [comment]
nice @doudar ! I didn't know this!

Maybe we can ask to the facebook group? Would you like to try? Let's just ask about if someone has a Wattbike (i never received a email from a Wattbike user)

**Reply by doudar** [reply]
Sure - I can post the same thing over on Facebook
